meta/conf/machine/salvator-x-m3.conf

```diff
@@ -0,0 +1,3 @@
+require xt-distro-machine.inc
+
+MACHINEOVERRIDES =. "rcar:rcar-gen3:"
```

Comment:
instead of having a single Salvator-X machine we now have salvator-x-h3 and salvator-x-m3 machines, thus we can tell what kind of SOM is installed into salvator-x board